### PR TITLE
added  missing items to feature

### PR DESCRIPTION
**Pull request by robserm**
I have added missing items like copyright and description to the feature.
Another missing item are required features. So now the user may define one or more requirements for a feature, see example below:

platform {
    feature(id: 'some_feature',
            name: '%feature.label',
            label: '%feature.label',
            provider: '%feature.providerName',
            license: '%license.url',
            copyright: "%feature.copyright",
            description: "%feature.description"
            )
    {
        requires (
            featureName: "required_feature",
            version: "10.2.0",
            match: "greaterOrEqual"
        )
    }
   //here are the dependencies...
}

### CODE REVIEW

File: src/main/groovy/org/standardout/gradle/plugin/platform/internal/config/ArtifactFeature.groovy [hunk 1]
```diff
@@ -203,6 +216,18 @@ class ArtifactFeature implements Feature {
 		def invokeMethod(String name, def args) {
 			//TODO support manually adding a feature reference
```
[the remark this answers is not in the document]
Comment (robserm) [reply src/main/groovy/org/standardout/gradle/plugin/platform/internal/config/ArtifactFeature.groovy]: removed , I had to miss that

File: build.gradle [hunk 2]
```diff
@@ -140,7 +140,12 @@ signing {
...
 
 tasks.withType(Sign) {
+    doFirst {
+        println "isRelease: $isRelease"
+        println "isCI: $isCI"
+    }
     onlyIf {
-        isRelease || isCI
+//        isRelease || isCI
+		false
```
[the remark this answers is not in the document]
Comment (robserm) [reply build.gradle]: ahhh, I am so sorry this change was not supposed to be a part of this pull request, now the original code is restored

**Comment by robserm** [comment]
I have also added a small explanation to README.md. I hope the content fits the rest of the document.

**Review by stempler — APPROVED** [review]

**Comment by stempler** [comment]
Included in the 3.1.0 release.

**Pull request merged**